Remotion/ObjectBinding/Web.Development.WebTesting/ScreenshotCreation/ScreenshotBocDateTimeValuePickerExtensions.cs

```diff
@@ -172,30 +172,51 @@ private static int GetDayOfTheWeekIndex (DateTime date)
       return (int) date.DayOfWeek - 1;
     }
 
-    private static bool IsVisible (IFluentScreenshotElementWithCovariance<ScreenshotBocDateTimeValuePicker> fluentDatePicker)
+    private static void WaitUntilVisible (IFluentScreenshotElementWithCovariance<ScreenshotBocDateTimeValuePicker> fluentDatePicker, int timeout)
     {
       var dateTimeValue = fluentDatePicker.Target.DateTimeValue;
-      var id = string.Join ("_", dateTimeValue.Scope.Id, "DatePicker");
-      var result = dateTimeValue.Context.RootScope.FindId (id, Options.NoWait);
+      var datePickerID = GetDatePickerID (dateTimeValue);
+      var seleniumDriver = (IWebDriver) dateTimeValue.Context.Browser.Driver.Native;
 
-      return result.Exists (Options.NoWait);
+      var webDriverWait = new WebDriverWait (seleniumDriver, TimeSpan.FromMilliseconds (timeout))
+                          {
+                              PollingInterval = TimeSpan.FromMilliseconds (300)

```

Comment:
Would it be better to use a division by e.g. 10 of the timeout instead of some arbitratry hardcoded value?